src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -569,6 +556,33 @@ private BoundLambda ReallyBind(NamedTypeSymbol delegateType)
             return result;
         }
 
+        private LambdaSymbol CreateLambdaSymbol(NamedTypeSymbol delegateType, Symbol containingSymbol, out MethodSymbol invokeMethod, out TypeWithAnnotations returnType, out DiagnosticBag diagnostics, out ReturnInferenceCacheKey cacheKey)
+        {
+            invokeMethod = DelegateInvokeMethod(delegateType);
+            returnType = DelegateReturnTypeWithAnnotations(invokeMethod, out RefKind refKind);
+            diagnostics = DiagnosticBag.GetInstance();
+            cacheKey = ReturnInferenceCacheKey.Create(delegateType, IsAsync);
+
+            // When binding for real (not for return inference), we cannot reuse a body of a lambda
```

Comment:
>// When binding for real (not for return inference), we cannot reuse a body of a lambda [](start = 12, length = 87)

It feels like this comment belongs in its original place in ```ReallyBind``` rather than in this method which doesn't do anything with body.